sematic/ui/src/components/LogPanel.tsx

```diff
@@ -37,7 +41,7 @@ export default function LogPanel(props: { run: Run }) {
       {run.exception_metadata_json && <Box sx={{ paddingBottom: 4, }} >
         <Exception exception_metadata={run.exception_metadata_json} /></Box>}
 
-      <Box sx={{ flexGrow: 1, display: 'flex', flexDirection: 'column'}} >
+      {!runHasException && <Box sx={{ flexGrow: 1, display: 'flex', flexDirection: 'column'}} >
```

Comment:
Is this saying we don't show logs if there's an exception? We still want to be able to view logs if there's an exception; it's actually especially important in that case.

Reply:
This ^

Reply:
Thanks for the feedback. Now I have changed it to squeezing all the error/exception messages into the scroll panel itself. 

This is better than putting the exception message outside and above the scroll area. If we do that, the scroll area will be super short.

If you have other UI design considerations, please kindly let me know.